### DISCUSSION TITLE
✅ Enhanced Search Element in Elementor

### DISCUSSION THREAD

**Discussion by ayoubkhan558** [elementor/elementor#28134] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

The current search element in Elementor does not support filtering for specific targets (products or posts,  custom posts) or AJAX product search, limiting its functionality and user experience.

### Describe the solution you'd like.

Enhance the search element by adding options to filter searches specifically for products or posts. Additionally, implement support for AJAX product search to provide a faster and more user-friendly experience.

### Describe alternatives you've considered.

Users currently need to use third-party search plugins to achieve these functionalities. These plugins can be complex to configure and may not integrate seamlessly with Elementor.

https://wordpress.org/plugins/relevanssi/
https://wordpress.org/plugins/add-search-to-menu/

### Additional context

Enhancing the search element with target-specific filtering and AJAX product search would significantly improve its functionality and user experience. This aligns with Elementor’s goal of providing comprehensive website building tools without relying on third-party plugins.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by nicholaszein** [comment]
Hello there @ayoubkhan558 !

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with Elementor Pro v3.23! 🥳 

You can now use the [new Search widget with Live Results](https://elementor.com/help/search-widget/), with support for custom queries.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
